### ISSUE TITLE
Alt Tags not being seen through site audit on multiple crawlers

### ISSUE THREAD

**Issue by highway9support** [nystudio107/craft-seomatic#1461]
Through Craft's asset menu, I have updated the alt text for all but about 50 images on the site (from a starting point of around 500), however through tools like SEMRush and Ahrefs, they note that there are still hundreds of images with missing alt text. Is there an different solution I need to take to updating the alt text for these images? Or maybe I'm doing something wrong?

Please advise
![Screenshot 2024-04-23 091255](https://github.com/nystudio107/craft-seomatic/assets/166417341/2cb2f9a4-d24d-41a8-b28b-97e274253022)

**Comment by khalwat** [comment]
You're not doing anything wrong, but just adding the alt text in the CMS won't cause that alt text to be displayed on the frontend.

You'd need to make changes to the Twig templates that output your images (this doesn't have anything to do with SEOmatic).
